tests/unit/helpers/mocks.py

```diff
@@ -156,3 +157,8 @@ class MockRole(object):
 
     def __init__(self, name):
         self.name = name
+
+
+@contextmanager
+def identity_context_manager(arg):
```

Comment:
I'd rename this to have the 'mock' word somewhere. I know it's in the `mocks` module, but once imported can mislead people.

Reply:
so just `mock_context_manager`?

Reply:
I'd use something more specific, like `mock_retrieve_uri`.